mscore/palette.cpp

```diff
@@ -136,12 +158,13 @@ bool Palette::filter(const QString& text)
                         }
                   if (c)
                         break;
-                  }
+                  }           
             if (t.isEmpty() || c)
                   dragCells.append(cell);
             bool contains = t.isEmpty() || c;
-            if (!contains)
+            if (!contains) {
```

Comment:
in general we don't use braces for single-line if's like this (also same thing a little bit above)